src/storage/engine/raftkv.rs

```diff
@@ -467,23 +470,35 @@ impl<S: RaftStoreRouter> Engine for RaftKv<S> {
 
 impl Snapshot for RegionSnapshot {
     fn get(&self, key: &Key) -> engine::Result<Option<Value>> {
+        fail_point!("raftkv_snapshot_get", |_| {
```

Comment:
can this hurt peroformance?

/cc @BusyJay

Reply:
Of course if build without `no-fail`.